src/openzaak/components/documenten/tests/test_validation.py

```diff
@@ -6,19 +6,18 @@
 
 from privates.test import temp_private_root
 from rest_framework import status
-from rest_framework.test import APITestCase
 from vng_api_common.tests import get_validation_errors, reverse, reverse_lazy
 from vng_api_common.validators import IsImmutableValidator
 
 from openzaak.components.catalogi.tests.factories import InformatieObjectTypeFactory
-from openzaak.utils.tests import JWTAuthMixin
+from openzaak.utils.tests import APITestCaseCMIS, JWTAuthMixin
 
 from ..constants import OndertekeningSoorten, Statussen
 from .factories import EnkelvoudigInformatieObjectFactory
 
 
 @override_settings(ALLOWED_HOSTS=["testserver"])
-class EnkelvoudigInformatieObjectTests(JWTAuthMixin, APITestCase):
+class EnkelvoudigInformatieObjectTests(JWTAuthMixin, APITestCaseCMIS):
```

Comment:
If I understand correctly, CMIS is enabled by default (at the moment), so all these tests switched from testing Django models to testing CMIS. Please copy this file with the postfix `_cmis` and set appropriate CMIS_ENABLED overrides on classes in this file, and on the copied file.